### ISSUE TITLE
"Re-Run All Tests" from Results UI

### ISSUE THREAD

**Issue by jimkring** [JKISoftware/Caraya#93]
During our Caraya 1.0 presentation/webinar @JamesMc86 mentioned it would be nice to have the ability to "re-run all tests" from the Results UI.

In some cases, this could be possible and I can also see how, in some cases this could be very challenging for the UI to know the context under which the tests were involved. Probably, the "Run Tests.vi" would need to pass some information into the GUI, so that the GUI is able to re-invoke the same test set-up.

FYI @francois-normandin

**Comment by francois-normandin** [comment]
We can already rerun tests.
![image](https://user-images.githubusercontent.com/11728548/82735787-4b71f680-9cf2-11ea-99cc-eca6baac8c4f.png)

However, the challenge right now is to rerun **and change test configuration**. For example, let's say we would like to rerun the tests, but with Verbose? flag toggled. Very hard to do with Test Manager and Test Engine coupled the way they are.

This should be feasible to add as a feature, but only after decoupling Test Engine and UI. Currently, I'm sure a workaround is workable, but I would suggest we keep this for a future update.
